Script/char.dat

```diff
@@ -8121,7 +8136,101 @@ bum
+    "\"Har har har, it's always funny when you smack somebody on the head hard enough that they get this stupid, but quite happy smile and completely forget what they were doing. Har har har!\"",
+    "\"I once managed to mirror myself a doppelganger! We were unstoppable in battle, but sadly he didn't last.\"",
+    "\"Maybe you've heard about those fabled wands of cloning? I can think about a few things I wouldn't mind having more of, har har har.\"",
+    "\"Don't believe those legends about grateful genies released from their lamps! I've met one so grumpy he tried to behead me just because I rubbed his lamp.\"",
+    "\"Adamant, huh? Some say it's truly miraculous metal, but it's really just a noble iron, but heavy! Har har har!\"",
+    "\"Breaking your wands can have quite an effect, har har har. I've learned the hard way.\"",
+    "\"One of my friends once told me that altars and wands of polymorph mix surprisingly well. Hm, I'll have to try for myself one day.\"",
+    "\"My mama always told me, eat carrots and you'll see as well as a giant golden eagle.\"",
+    "\"Mushrooms are my favourite! They spawn so quickly you'll never run out of food! Har har har!\"",
+    "\"My mama always told me that fat people are not very agile and gaunt people are weak. It's best to try and stay fit, she said.\"",
+    "\"Ommels, eh? Magnificent creatures. Some say they have bones harder than steel and teeth like adamant!\"",
+    "\"Ommels, eh? Magnificent creatures. Some say you can even make good use of their excretions.\"",
+    "\"If you never had some troll blood, you should try it. It's so sweet, so tasty and nutritious... Enough to make a man sick.\"",
+    "\"Trolls really have tough hides. I made a pair of boots from the one I encountered in mountains far to the north, and used them for years before they wore out.\"",
+    "\"One priestess once told me that angels look out for ardent worshippers. I'm not sure what ardent means, so I just smiled and nodded.\"",
+    "\"Those kamikaze dwarves can't tell friend from foe. Maybe the gods don't care about a little friendly fire? Har har har!\"",
+    "\"Have you seen Petrus? Wise men have massive beards, indeed. Har har har!\"",
+    "\"One of my friends always said that when everything else fails, you can always bite your foes to death. Granted, she was a werewolf, but still...\"",
+    "\"Don't believe those rumors about a valpurium flaming sword +10 just lying around! It's all lies! Lies, I say!\"",
+    "\"I've heard tales about arcanite, a magical metal lighter than some wood! I would love to try out a shield made of arcanite.\"",
+    "\"One priestess once told me that phoenix feathers rise from their ashes. I wonder how incredible the whole birds must be, then.\"",
+    "\"Be careful about the minefields deep underground. The Dwarven Wars left behind so many landmines it's smart to invest into some fireproof dragon hide and a ring of fire resistance before going anywhere.\"",
```

Comment:
It's a hint that dragon hide is nearly fireproof, the most fire-resistant cloth material there is.